### PR TITLE
[Merged by Bors] - feat: Add `IsAlgebraic.inv_iff`.

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Algebraic.lean [hunk 1]
```diff
@@ -193,6 +193,29 @@ theorem Transcendental.pow {r : A} (ht : Transcendental R r) {n : ℕ} (hn : 0 <
     Transcendental R (r ^ n) := fun ht' ↦ ht <| ht'.of_pow hn
 #align transcendental.pow Transcendental.pow
 
+lemma IsAlgebraic.invOf {x : S} [Invertible x] (h : IsAlgebraic R x) : IsAlgebraic R (⅟ x) := by
+  obtain ⟨p, hp, hp'⟩ := h
+  refine ⟨p.reverse, by simpa using hp, ?_⟩
+  rwa [Polynomial.aeval_def, Polynomial.eval₂_reverse_eq_zero_iff, ← Polynomial.aeval_def]
+
+lemma IsAlgebraic.invOf_iff {x : S} [Invertible x] :
```
Comment (joelriou) [inline Mathlib/RingTheory/Algebraic.lean:201]: I would suggest this (and similarly for `IsAlgebraic.inv_iff`):
```suggestion
lemma IsAlgebraic.invOf_iff (x : S) [Invertible x] :
```

**Review by joelriou — APPROVED** [review]
I have made some small suggestions.

bors d+

File: Mathlib/RingTheory/Algebraic.lean (hunk 1 above)
Comment (erdOne) [reply Mathlib/RingTheory/Algebraic.lean:201]: I think these iffs usually have implicit arguments?

**Comment by erdOne** [comment]
bors merge

**Pull request closed without merging**